### PR TITLE
Homework. JDBC implementation

### PR DESCRIPTION
**Pull request by Fedia23**
The original task was:
Your task is to integrate storage of emails from previous task with SQL databases using both pure JDBC API and Hibernate. You should create a generic interface with all storage operations you do and then provide two implementations for it: one using JDBC and one using Hibernate. Please do not keep results in local collection field inside your implementation anymore. If you need to get data from db - do the query!

### CODE REVIEW

File: src/main/java/com/example/demo/dao/impl/JdbcConnection.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    @Override
+    @SneakyThrows
+    public void deleteById(Long id) {
+        PreparedStatement statement = this.getDbConnection().prepareStatement(SqlRequestMessages.SQL_DELETE_MESSAGE_ID.getRequest());
+        statement.setLong(1, id);
+        statement.execute();
+        transaction(statement);
+    }
+
+    private void transaction(PreparedStatement statement) throws SQLException {
```
Comment (vova-pereb) [inline src/main/java/com/example/demo/dao/impl/JdbcConnection.java]: Give more meaningful name for this method.

File: src/main/java/com/example/demo/model/enums/jdbc/JdbcConnect.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.example.demo.model.enums.jdbc;
+
+
+public enum JdbcConnect {
```
Comment (vova-pereb) [inline src/main/java/com/example/demo/model/enums/jdbc/JdbcConnect.java]: Configuration classes should not be represent as enum.
It is not a list of similar items.

File: src/main/java/com/example/demo/validations/DateValidation.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.example.demo.validations;
+
+import java.util.Date;
+
+public class DateValidation {
+
+    public DateValidation() {
```
Comment (vova-pereb) [inline src/main/java/com/example/demo/validations/DateValidation.java]: Redundant constructor.

File: src/main/java/com/example/demo/validations/DateValidation.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    }
+
+    public static boolean dateTransformer(long date, long jdate) {
+        Date current = new Date();
+        return jdate + (date * 1000) <= current.getTime() ? true : false;
```
Comment (vova-pereb) [inline src/main/java/com/example/demo/validations/DateValidation.java]: Can be simplified.

File: src/main/java/com/example/demo/dao/impl/JdbcConnection.java (hunk 1 above)
Comment (Fedia23) [reply src/main/java/com/example/demo/dao/impl/JdbcConnection.java]: Done

File: src/main/java/com/example/demo/model/enums/jdbc/JdbcConnect.java (hunk 2 above)
Comment (Fedia23) [reply src/main/java/com/example/demo/model/enums/jdbc/JdbcConnect.java]: Done

File: src/main/java/com/example/demo/validations/DateValidation.java (hunk 3 above)
Comment (Fedia23) [reply src/main/java/com/example/demo/validations/DateValidation.java]: Done

File: src/main/java/com/example/demo/validations/DateValidation.java (hunk 4 above)
Comment (Fedia23) [reply src/main/java/com/example/demo/validations/DateValidation.java]: Done

**Pull request closed without merging**